libs/spark/src/NavBarButton.tsx

```diff
@@ -3,16 +3,22 @@ import { withStyles } from '@material-ui/core/styles';
 import { Button, ButtonProps } from '@material-ui/core';
 import { theme } from './styles';
 
-export type NavBarButtonProps = Omit<ButtonProps, 'variant' | 'color' | 'size'>;
+export type NavBarButtonProps = Omit<ButtonProps, 'variant' | 'color'>;
 
 export const NavBarButton = withStyles({
   root: {
-    height: 48,
     borderRadius: 8,
+    backgroundColor: 'transparent',
+    border: '2px solid transparent',
     padding: 8,
```

Comment:
since border is 2px, padding has to account for that
```suggestion
    padding: 6,
```